spec/mouse.spec.js

```diff
@@ -184,83 +185,110 @@ describe('handling mouse in static objects', function() {
 
         it('in/out events properly work in sequences', function() {
 
             fireCanvasEvent('mousemove', 25, 75);
-            expect(log.join(MARKER)).toEqual('root: mouseenter@25;30 -> e2' + MARKER +
-                                             'e2: mouseenter@25;30 -> e2');
+            expect(log.stringify(MARKER)).toEqual([ 'root: mouseenter@25;30 -> e2',
+                                                    'e2: mouseenter@25;30 -> e2' ].join(MARKER));
 
-            log = [];
+            log.clear();
 
             fireCanvasEvent('mousemove', 25, 75);
-            expect(log.join(MARKER)).toEqual('');
+            expect(log.stringify(MARKER)).toEqual('');
 
             fireCanvasEvent('mousemove', 26, 76);
-            expect(log.join(MARKER)).toEqual('');
+            expect(log.stringify(MARKER)).toEqual('');
 
             fireCanvasEvent('mousemove', 25, 25);
-            expect(log.join(MARKER)).toEqual('e2: mouseexit@null;null -> e2' + MARKER +
-                                             'e1: mouseenter@25;25 -> e11' + MARKER +
-                                             'e11: mouseenter@25;25 -> e11');
+            expect(log.stringify(MARKER)).toEqual([ 'e2: mouseexit@null;null -> e2',
+                                                    'e1: mouseenter@25;25 -> e11',
+                                                    'e11: mouseenter@25;25 -> e11' ].join(MARKER));
 
-            log = [];
+            log.clear();
 
             fireCanvasEvent('mousemove', 76, 6);
-            expect(log.join(MARKER)).toEqual('e11: mouseexit@null;null -> e11' + MARKER +
-                                             'e12: mouseenter@1;1 -> e12');
+            expect(log.stringify(MARKER)).toEqual([ 'e11: mouseexit@null;null -> e11',
+                                                    'e12: mouseenter@1;1 -> e12' ].join(MARKER));
 
-            log = [];
+            log.clear();
 
             fireCanvasEvent('mousemove', 25, 75);
-            expect(log.join(MARKER)).toEqual('e12: mouseexit@null;null -> e12' + MARKER +
-                                             'e1: mouseexit@null;null -> e12' + MARKER +
-                                             'e2: mouseenter@25;30 -> e2');
+            expect(log.stringify(MARKER)).toEqual([ 'e12: mouseexit@null;null -> e12',
+                                                    'e1: mouseexit@null;null -> e12',
+                                                    'e2: mouseenter@25;30 -> e2' ].join(MARKER));
+
+        });
+
+    });
+
+    describe('other types of events', function() {
+
+        var MARKER = '\n';
+
+        var targets = [ root, e1, e2, e11, e12 ];
+        var events = [ 'mousemove', 'mousedown', 'mouseup', 'mousedoubleclick' ];
+
+        var log = new EventLog(targets, events);
+
+        beforeEach(function() {
+            log.subscribe();
+        });
+
+        afterEach(function() {
+            log.unsubscribe();
+            log.clear();
+        });
 
```

Comment:
I believe we should drop this "target", there's no need for it now, we're not pumping events through like, event bus or somethings

Let's make it barely sufficient for now